src/locales/en/common.json

```diff
@@ -1,12 +1,26 @@
 {
-    "header":{
-        "category":{
-            "men":"Men",
-            "women":"Women",
-            "electronics": "Electronics"
-        }
+  "header": {
+    "category": {
+      "men": "Men",
+      "women": "Women",
+      "electronics": "Electronics"
     },
-    "home":{
-        
+    "user": {
+      "signup": "Signup",
+      "login": "Login"
+    },
+    "categorytype": {
+      "inthespotlight": {
+        "title": "IN THE SPOTLIGHT",
```

Comment:
Don't do the hard case here. Use `toUpperCase` when rendering in the component.

Reply:
okay